ruby/ql/src/queries/security/cwe-020/MissingFullAnchor.qhelp

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE qhelp PUBLIC
+"-//Semmle//qhelp//EN"
+"qhelp.dtd">
+<qhelp>
+
+	<overview>
+		<p>
+            Regular expressions in Ruby can use anchors to match the beginning and end of a string. 
+            However, if the <code>^</code> and <code>$</code> anchors are used, 
+            the regular expression can match a single line of a multi-line string.
```

Comment:
```suggestion
            the regular expression can match a single line of a multi-line string. This allows bad actors to bypass your regular expression checks and execute code using a multi-line string.
```

Suggesting this sentence to provide a little more clarity on why this poses a security risk

Reply:
It does not necessarily allow for code-execution. 
I'll try to adjust the sentence to add more clarity.

Reply:
Ah, gotcha 👍🏼 the sentence you've added looks good to me!